.ci/scripts/build.sh

```diff
@@ -160,21 +153,13 @@ fi
 echo "Calling make"
 echo $CXX
 echo $CC
-echo make ${target:-daal_c} ${make_op} \
+echo make ${target:-daal} ${make_op} \
     COMPILER=${compiler} \
-    REQCPU="${optimizations}" \
-    BACKEND_CONFIG="${backend_config}" \
-    PLAT=$PLAT
-make ${target:-daal_c} ${make_op} \
+    REQCPU=\"${optimizations}\" \
+    BACKEND_CONFIG=${backend_config} \
+    PLAT=${PLAT}
+make ${target:-daal} ${make_op} \
     COMPILER=${compiler} \
     REQCPU="${optimizations}" \
-    BACKEND_CONFIG="${backend_config}" \
+    BACKEND_CONFIG=${backend_config} \
```

Comment:
any reason for reversing order here?

Reply:
I can't see that the order was reversed, but I did lose some quotes. I put those back in, and set the default target to daal instead of daal_c

Reply:
Should we put onedal_c ?

Reply:
Happy to change it. Also, no idea what the escaping was that I added, so reverted back

Reply:
Looks good to me

Reply:
Functionally daal_c vs. onedal_c should be fine either way since its just default value but in cases where script is called we pass target arg so that is used instead